src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -147,37 +145,21 @@ class BaseCreateCourseForm extends React.Component {
           <hr />
           <Field
             name="start"
-            type="date"
-            component={RenderInputTextField}
-            format={value => getDateString(value)}
-            normalize={value => moment.utc(value).format(DATE_FORMAT)}
-            label={
-              <FieldLabel
-                id="start-label"
-                text="Start date"
-                required
-                helpText={startDateHelp}
-              />
-            }
-            placeholder="mm/dd/yyyy"
-            required
+            component={DateTimeField}
+            dateName="startDate"
```

Comment:
`dateName` and `timeName` feel like strings that `DateTimeField` could generate itself. Just take the `name` value and append/prepend `date` and `time`.